motor/core.py

```diff
@@ -544,7 +544,7 @@ class AgnosticDatabase(AgnosticBaseProperties):
     drop_collection = AsyncCommand().unwrap("MotorCollection")
     get_collection = DelegateMethod().wrap(Collection)
     list_collection_names = AsyncRead(doc=docstrings.list_collection_names_doc)
-    list_collections = AsyncRead()
+    list_collections = AsyncRead().wrap(CommandCursor)
```

Comment:
Is this a bug fix? If so can you open a new motor bug ticket so we can remember to mention this fix in the changelog?